### ISSUE TITLE
Period::getDays() returns only days

### ISSUE THREAD

**Issue by pupitooo** [brick/date-time#11]
```php
// expected
$until = LocalDate::of(2018, 11, 1);
$today = LocalDate::of(2018, 9, 1);
$period = $today->until($until);
$days = $period->getDays(); // 1

$until = LocalDate::of(2018, 11, 1);
$today = LocalDate::of(2018, 9, 1);
$days = $until->toEpochDay() - $today->toEpochDay(); // 62
```
I miss any funcion to get real days. It's not possible to get real days from Period.

**Comment by BenMorel** [comment]
Hi, Period cannot return the total number of days, as it depends on the year/month it's applied to.

For example:

- 2018-01-15 + a period of 1 month = 2018-02-15 = 31 days
- 2018-02-15 + a period of 1 month = 2018-03-15 = 28 days

Even though your use case is legitimate, it therefore does not belong to Period.

Your approach is the correct one:

```php
$days = $until->toEpochDay() - $today->toEpochDay();
```

Do you feel like there should be a function such as `LocalDate::daysUntil(LocalDate $date) : int`?

**Comment by BenMorel** [comment]
I forgot to mention that this is because `until()` calculates the number of years, months and days. The alternative would've been to return a Period with only the `days` set, and zero `years` and `months`.

I checked what Java does, and [they provide 2 methods](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html):

- `Period until(ChronoLocalDate endDateExclusive)`
  > Calculates the period between this date and another date as a Period.
- `long until(Temporal endExclusive, TemporalUnit unit)`
  > Calculates the amount of time until another date in terms of the specified unit.

The first method does the same as the current implementation, but it looks like the second method can be used to get the result in terms of years, months or days.

I'm not sure we want to follow this generic route, but I'm not against adding a method like `LocalDate::daysUntil()`. Thoughts?

The alternative would be to add an optional `bool` parameter to `until()`, to allow it to return a Period that only contains days. I'm not sure what this is worth.

**Comment by pupitooo** [comment]
Just now I am using approach
```php
$days = $until->toEpochDay() - $today->toEpochDay();
```
but it contains counting in my code. And this is not so easy to use (you have to think about order of items).
I preffer `LocalDate::daysUntil()` because bool parameter switch behaviour of all result. Separated function will be little bit more transparent.

**Comment by BenMorel** [comment]
Implemented as `LocalDate::daysUntil()` as released as version [0.1.9](https://github.com/brick/date-time/releases/tag/0.1.9)!
